libshvcore/src/utils/shvlogfilereader.cpp

```diff
@@ -91,7 +91,9 @@ bool ShvLogFileReader::next()
 		cp::RpcValue st = row.value(Column::ShortTime);
 		m_currentEntry.shortTime = st.isInt() && st.toInt() >= 0? st.toInt(): ShvJournalEntry::NO_SHORT_TIME;
 		m_currentEntry.domain = row.value(Column::Domain).toString();
-		m_currentEntry.sampleType = static_cast<ShvJournalEntry::SampleType>(row.value(Column::SampleType).toUInt());
+		if (row.size() - 1 >= Column::SampleType) {
+			m_currentEntry.sampleType = static_cast<ShvJournalEntry::SampleType>(row.value(Column::SampleType).toUInt());
```

Comment:
co tahle zmena dela, je to delsi zpusob, jak prepsat ten zmeneny radek, jestli jsem neco neprehledl